### DISCUSSION TITLE
Accessing Generated Solution Projects

### DISCUSSION THREAD

**Discussion by chrisbewz** [nuke-build/nuke#1496] [Q&A]
I've noticed that in some solutions I’ve worked on, projects sometimes appear under the `PrimitiveProjects` property of the `Solution` class, along with solution folders, instead of the `Projects` property. Because of this, `GetProject` calls may return null for a project name expected to be found on solution as it only searches for valid items under `Projects` property.

Am I missing something about how projects are scaffolded from the solution, or could this be related to how I’m including projects in my solution?

**Comment by matkoch** [comment]
This is all very vague. You should get more specific and show some code.
